apix-integrationtests-client/alfresco/build.gradle

```diff
@@ -0,0 +1,90 @@
+def integrationTestsProjectDir = project.projectDir
+allprojects {
+    // Subproject of supported Alfresco versions
+    def subproject_alfresco_version = project.projectDir.name == "alfresco"
+            ? "231" // minimum supported version
+            : project.projectDir.name
+
+    // Following needs to apply to shared (./src) project and version-specific (e.g. ./70) projects.
+    apply plugin: 'idea'
+    apply plugin: 'java-library'
+    apply plugin: 'eu.xenit.alfresco'
+    apply plugin: 'eu.xenit.docker-alfresco'
+    apply from: "${rootProject.projectDir}/alfresco/${subproject_alfresco_version}/overrides.gradle"
+
+    sourceSets {
+        main.java.srcDirs = []
+        main.resources.srcDirs = []
+        integrationTest {
+            java.srcDirs = ["${integrationTestsProjectDir}/src/main/java"]
+            resources.srcDirs = ["${integrationTestsProjectDir}/src/main/resources"]
+            compileClasspath += sourceSets.main.output + configurations.testRuntimeClasspath
+            runtimeClasspath += sourceSets.main.output + configurations.testRuntimeClasspath
+        }
+    }
+
+    configurations {
+        integrationTestImplementation.extendsFrom testImplementation
+        integrationTestRuntimeOnly.extendsFrom testRuntimeOnly
+    }
+
+    task integrationTest(type: Test) {
+        description = 'Runs the integration tests.'
+        group = 'verification'
+        testClassesDirs = sourceSets.integrationTest.output.classesDirs
+        classpath = sourceSets.integrationTest.runtimeClasspath
+        shouldRunAfter test
+    }
+    check.dependsOn integrationTest
+
+    dependencies {
+        alfrescoProvided(project(":apix-rest-v1")) { transitive = false }
+        alfrescoProvided(project(":apix-interface")) { transitive = false }
+        alfrescoProvided(project(":apix-interface")) { transitive = false }
+
+        alfrescoProvided platform("org.alfresco:acs-community-packaging:${alfresco_version}")
+        alfrescoProvided("org.alfresco:alfresco-repository")
+        alfrescoProvided('org.alfresco:alfresco-remote-api')
+        alfrescoProvided('org.alfresco:alfresco-data-model')
+        testImplementation group: 'org.mockito', name: 'mockito-core', version: '5.0.0'
+        testImplementation("org.apache.httpcomponents:fluent-hc:${http_version}") {
+            exclude group: 'commons-logging'
+            exclude group: 'org.apache.httpcomponents', module: 'httpcore'
+        }
+        // Add services used to the integration test fatjar, since we can't access the ones deployed in Alfresco
+        testImplementation (project(":apix-impl:apix-impl-${subproject_alfresco_version}")) {
+            // Already includes apix-interface, we need to exclude it to avoid Linkage errors from
+            // having 2 of the same classes on the classpath
+            // (1 in Alfresco from apix-impl AMP + 1 in DE from the integration tests fat jar)
```

Comment:
@codingBenVdS this is not needed anymore since we don't have DE?

Reply:
Should be the case here. Removing and testing.